tests/features/asset_distribution/add_solution.feature

```diff
@@ -0,0 +1,53 @@
+@api
+Feature: "Add asset distribution" visibility options.
+  In order to manage distributions
+  As a moderator
+  I need to be able to add "Asset distribution" rdf entities through UI.
+
+  Scenario: "Add distribution" button should only be shown to moderators.
+    Given the following collection:
+      | name | Asset Distribution Test                  |
+      | uri  | https://a.distribution/solution/add/test |
+      | logo | logo.png                                 |
+    And the following solution:
+      | name              | Asset random name                        |
+      | uri               | http://joinup.eu/ad/solution/api/bar     |
+      | description       | Some reusable random description         |
+      | elibrary creation | 1                                        |
+      | groups audience   | https://a.distribution/solution/add/test |
+
+    When I am logged in as a "moderator"
+    And I go to the homepage of the "Asset random name" solution
```

Comment:
How do I go to the homepage of the "Asset random name" solution? As a UAT tester this is not clear. It would be better to say where to click in the interface.